aptos-move/vm-genesis/src/lib.rs

```diff
@@ -259,6 +265,10 @@ fn create_and_initialize_main_accounts(
             MoveValue::Bool(allow_new_validators),
             MoveValue::U64(rewards_rate_numerator),
             MoveValue::U64(rewards_rate_denominator),
+            MoveValue::Bool(enable_on_chain_governance),
```

Comment:
feels like we should just initialize on chain governance directly

Reply:
also because we're directly calling into this, we can also make a struct out of this and uses options to signify if it enabled or not.